ansible/roles/util-cfg-service/templates/terrain.properties.j2

```diff
@@ -89,7 +89,6 @@ terrain.garnish.type-attribute = ipc-filetype
 
 # File IO settings
 terrain.fileio.url-import-app = 1E8F719B-0452-4D39-A2F3-8714793EE3E6
-terrain.fileio.max-edit-file-size = {{ max_edit_file_size }}
```

Comment:
As we discussed at one point, I'm letting nginx handle this from here on in. This was only enforced for this endpoint anyway (not upload or saveas) and nginx is almost certainly better at it.